signer-service/src/api/services/pendulum.service.js

```diff
@@ -108,14 +126,26 @@ exports.sendStatusWithPk = async () => {
       if (remainingMaxSubsidiesAvailable.lt(SUBSIDY_MINIMUM_RATIO_FUND_UNITS)) {
         isTokensSufficient = false;
         console.log(`Token ${token} balance is insufficient.`);
+
+        slackNotifier.sendMessage({
+          text: `Current balance of funding account is ${nativeToDecimal(
+            remainingMaxSubsidiesAvailable,
```

Comment:
`remainingMaxSubsidiesAvailable` is actually not a currency amount but just a number counting how many subsidies are still possible with the remaining funds, assuming that every subsidy requires the maximum amount. Therefore `nativeToDecimal` does not make sense here.

Or you use `tokenBalance` instead of `remainingMaxSubsidiesAvailable`.